integration/auth/user_mgt_test.go

```diff
@@ -133,7 +133,16 @@ func TestListUsers(t *testing.T) {
 		if _, ok := newUsers[u.UID]; ok {
 			count++
 			if u.PasswordHash == "" {
-				t.Errorf("Users() PasswordHash = empty; want = non-empty")
+				t.Errorf(
+					"Users() PasswordHash = empty; want = non-empty. A common cause would be " +
```

Comment:
Nit: Extract the "A common cause..." part into a constant.

Reply:
Personally, I (slightly) prefer inlining it here. But I've already refactored in c# (due to the linter not liking multi-line parameters) and in python (due to 4x repetition) so at this point, refactoring everywhere *is* starting to feel more natural. Done.